src/components/nokkelinformasjon/container/NokkelinformasjonContainer.tsx

```diff
@@ -49,18 +49,10 @@ export const NokkelinformasjonSide = () => {
   useEffect(() => {
     dispatch(hentLedere(fnr));
     dispatch(hentSykmeldinger(fnr));
-    if (fnr) {
-      dispatch(hentOppfoelgingsdialoger(fnr));
-    }
-  }, []);
-
-  useEffect(() => {
+    dispatch(hentOppfoelgingsdialoger(fnr));
     dispatch(hentOppfolgingstilfellerPersonUtenArbeidsiver(fnr));
-  }, [sykmeldingerState]);
-
-  useEffect(() => {
     dispatch(hentOppfolgingstilfelleperioder(fnr));
-  }, [ledereState, sykmeldingerState]);
+  }, [dispatch, fnr]);
```

Comment:
Man burde vel fortsatt reagere på endringer på `ledereState`? `oppfolgingstilfelleperioder` bruker ledere for å vite hva den skal hente.

Reply:
Hm, skjønner. Legger den tilbake :)